src/main/js/main_html5.js

```diff
@@ -418,6 +413,19 @@ require("../../../html5-common/js/utils/environment.js");
     };
 
     /**
+     * Prepares a video element to be played via API.  This is called on a user click event, and is used in
+     * preparing HTML5-based video elements on devices.  To prepare the element for playback, call pause and
```

Comment:
The comments says pause and play, but the ordering is play and pause.
Let's check all other similar  commenting as well